models/participant.yaml

```diff
@@ -0,0 +1,23 @@
+id: https://bican.org/examples/participant
+name: participant
+prefixes:
+  linkml: https://w3id.org/linkml/
+imports:
+  - linkml:types
+default_range: string
+
+classes:
+  Participant:
```

Comment:
i had a chance to discus this with a few folks and there are couple of considerations that should be added to this. 
- age concepts vary by species
- there are some components that should not change (datetime of birth) and some that could change (sex, weight, family, etc.,.)
- we need to add concepts like littermates not just lineage, which is what relatedparticiant was focusing on. this could be a role added to relatedparticipant.

Reply:
Also, some ages are measured from conception, some from birth -- some measured in days, some in weeks, some in years, etc.

Reply:
@patrick-lloyd-ray - we think date of birth should capture most scenarios but is a sensitive field for humans. in dandi we use gestational reference and birth reference as two reference frames, and we are using the duration format for ISO8601 to allow for any age representation.  

however age as a duration is also limited across species which have stages. also, we had to invent a format for age range (based on ISO8601), when precise age is not available. age is a complicated concept ;)